tools/debugging/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.10.0 AS builder
```

Comment:
Why have this file here if we've already put it in README?

Reply:
Because we use this file in ```docker build``` command, you must change it and use as a default behavior